Python package set major updates

FRidh opened the pull request:
###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Commit:
python.pkgs.pyqt5: 5.10.1 -> 5.11.3

Commit:
python.pkgs.sip: 4.19.8 -> 4.19.13

FRidh committed:
python: sphinxcontrib-httpdomain: 1.5.0 -> 1.7.0

FRidh committed:
python: sphinxcontrib-spelling: 2.2.0 -> 4.2.0

FRidh committed:
python: smartdc: 0.1.12 -> 0.2.0

FRidh committed:
python: sphinx_rtd_theme: 0.2.5b2 -> 0.4.2

FRidh committed:
python: smmap2: 2.0.4 -> 2.0.5

FRidh committed:
python: sphinxcontrib-openapi: 0.3.0 -> 0.3.2

FRidh committed:
python: sockjs-tornado: 1.0.5 -> 1.0.6

FRidh committed:
python: SPARQLWrapper: 1.7.6 -> 1.8.2

FRidh committed:
python: sorl-thumbnail: 11.12 -> 12.5.0

FRidh committed:
python: sopel: 6.3.1 -> 6.5.3

FRidh committed:
python: sphinxcontrib-plantuml: 0.7 -> 0.12

FRidh committed:
python: sqlparse: 0.2.2 -> 0.2.4

FRidh committed:
python: sqlmap: 1.2.10 -> 1.2.11

FRidh committed:
python: tempora: 1.13 -> 1.14

FRidh committed:
python: testpath: 0.3 -> 0.4.2

FRidh committed:
python: taskw: 1.0.3 -> 1.2.0

FRidh committed:
python: SQLObject: 3.3.0 -> 3.7.0

FRidh committed:
python: testscenarios: 0.4 -> 0.5.0

FRidh committed:
python: subprocess32: 3.5.2 -> 3.5.3

FRidh committed:
python: tqdm: 4.25.0 -> 4.28.1

FRidh committed:
python: tvdb_api: 1.10 -> 2.0

FRidh committed:
python: transitions: 0.6.8 -> 0.6.9

FRidh committed:
python: thumbor-pexif: 0.14 -> 0.14.1

FRidh committed:
python: tvnamer: 2.4 -> 2.5

FRidh committed:
python: thrift: 0.9.3 -> 0.11.0

FRidh committed:
python: tlslite: 0.4.8 -> 0.4.9

FRidh committed:
python: tox: 3.2.1 -> 3.5.3

FRidh committed:
python: toposort: 1.1 -> 1.5

FRidh committed:
python: tifffile: 0.15.1 -> 2018.10.18

FRidh committed:
python: tilestache: 1.50.1 -> 1.51.13

FRidh committed:
python: trezor_agent: 0.9.0 -> 0.9.3

FRidh committed:
python: Twisted: 18.7.0 -> 18.9.0

FRidh committed:
python: Twiggy: 0.4.5 -> 0.4.7

FRidh committed:
python: twitter: 1.15.0 -> 1.18.0

FRidh committed:
python: urllib3: 1.23 -> 1.24.1

FRidh committed:
python: virtualenv: 16.0.0 -> 16.1.0

FRidh committed:
python: uritemplate.py: 3.0.0 -> 3.0.2

FRidh committed:
python: virtualenvwrapper: 4.3 -> 4.8.2

FRidh committed:
python: vmprof: 0.3.3 -> 0.4.12

FRidh committed:
python: vidstab: 0.1.5 -> 1.0.0

FRidh committed:
python: txrequests: 0.9.2 -> 0.9.6

FRidh committed:
python: w3lib: 1.17.0 -> 1.19.0

FRidh committed:
python: word2vec: 0.9.1 -> 0.10.2

FRidh committed:
python: watchdog: 0.8.3 -> 0.9.0

FRidh committed:
python: web.py: 0.37 -> 0.39

FRidh committed:
python: xattr: 0.7.8 -> 0.9.6

FRidh committed:
python: willow: 0.2.2 -> 1.1

FRidh committed:
python: websocket_client: 0.51.0 -> 0.54.0

FRidh committed:
python: wsproto: 0.11.0 -> 0.12.0

FRidh committed:
python: websockify: 0.7.0 -> 0.8.0

FRidh committed:
python: waitress: 1.0.2 -> 1.1.0

FRidh committed:
python: wptserve: 1.4.0 -> 2.0

FRidh committed:
python: xkcdpass: 1.4.2 -> 1.16.5

FRidh committed:
python: whitenoise: 4.0 -> 4.1

FRidh committed:
python: webtest: 2.0.20 -> 2.0.32

FRidh committed:
python: WebOb: 1.7.3 -> 1.8.3

FRidh committed:
python: ZODB: 5.4.0 -> 5.5.1

FRidh committed:
python: xlsx2csv: 0.7.2 -> 0.7.4

FRidh committed:
python: zbase32: 1.1.2 -> 1.1.5

FRidh committed:
python: ZConfig: 3.2.0 -> 3.3.0

FRidh committed:
python: Yapsy: 1.11.223 -> 1.12.0

FRidh committed:
python: zfec: 1.4.24 -> 1.5.3

FRidh committed:
python: zdaemon: 4.0.0 -> 4.3

FRidh committed:
python: zope.configuration: 4.0.3 -> 4.3.0

FRidh committed:
python: xml2rfc: 2.10.3 -> 2.12.3

FRidh committed:
python: XStatic-jQuery-File-Upload: 9.22.0.1 -> 9.23.0.1

FRidh committed:
python: zdaemon: 4.0.0 -> 4.3

FRidh committed:
python: XStatic-Bootstrap: 3.3.7.1 -> 4.1.3.1

FRidh committed:
python: zope.component: 4.2.1 -> 4.5

FRidh committed:
python: xlrd: 0.9.4 -> 1.1.0

FRidh committed:
python: zope.dottedname: 3.4.6 -> 4.3

FRidh committed:
python: zope.proxy: 4.1.6 -> 4.3.1

FRidh committed:
python: zope.contenttype: 4.0.1 -> 4.4

FRidh committed:
python: zope.event: 4.0.3 -> 4.4

FRidh committed:
python: zope.interface: 4.5.0 -> 4.6.0

FRidh committed:
python: zope.i18nmessageid: 4.0.3 -> 4.3.1

FRidh committed:
python: zope.copy: 4.1.0 -> 4.2

FRidh committed:
python: zope.filerepresentation: 3.6.1 -> 4.2.0

FRidh committed:
python: zope.schema: 4.4.2 -> 4.9.3

FRidh committed:
python: zope.i18n: 3.8.0 -> 4.6.1

FRidh committed:
python: zope.location: 4.0.3 -> 4.2

FRidh committed:
python: zope.lifecycleevent: 3.7.0 -> 4.3

FRidh committed:
python: zope.deprecation: 4.1.2 -> 4.3.0

FRidh committed:
python: zope.exceptions: 4.0.8 -> 4.3

FRidh committed:
python: zope.testrunner: 4.8.1 -> 4.9

FRidh committed:
python: zope.size: 3.5.0 -> 4.3

FRidh committed:
python: zope.testing: 4.6.1 -> 4.7

FRidh committed:
pythonPackages.click: fix fetching

FRidh committed:
pythonPackages.pysocks: fix fetching

GrahamcOfBorg commented:
<!--REQUEST_ID=fd823ed4-3cbd-432a-b89a-8053d2a349e3-->
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49741&attempt_id=36091fb3-d36f-42bc-b409-138c5bae8cee)

Attempted: python

<details><summary>Partial log (click to expand)</summary><p>

```
copying path '/nix/store/3jnjp6g1ysdw7x754283rxc6cb8560qy-ncurses-6.1' from 'https://cache.nixos.org'...
copying path '/nix/store/975x162f3nm3ac13nfqn99rxdxfb2522-openssl-1.0.2p' from 'https://cache.nixos.org'...
copying path '/nix/store/251f3hjyih24ravx8ddfss9vwfm6dvq0-zlib-1.2.11' from 'https://cache.nixos.org'...
copying path '/nix/store/pjv42l4h1r0y3pkcqgbbwb2scbh7gwsg-acl-2.2.52' from 'https://cache.nixos.org'...
copying path '/nix/store/9vz0sgklpmr1m5s80yzys3ml9nlymqk1-sqlite-3.24.0' from 'https://cache.nixos.org'...
copying path '/nix/store/wfp2ypkspcx4a7ph0awa5kwy9vjwswjh-coreutils-8.30' from 'https://cache.nixos.org'...
copying path '/nix/store/1kfn90226didqra7lbwsdnhcjgps78jf-readline-6.3p08' from 'https://cache.nixos.org'...
copying path '/nix/store/ay8jfqdy1g7ldi77b1bd26hbhw6z8ndm-db-5.3.28' from 'https://cache.nixos.org'...
copying path '/nix/store/2vwrp5xwlkl9r8jiaxj85lxiv7b8kp4j-python-2.7.15' from 'https://cache.nixos.org'...
/nix/store/2vwrp5xwlkl9r8jiaxj85lxiv7b8kp4j-python-2.7.15
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=fd823ed4-3cbd-432a-b89a-8053d2a349e3-->
Success on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49741&attempt_id=dad6ba9d-a363-4231-8ddf-01805085dbb5)

Attempted: python

<details><summary>Partial log (click to expand)</summary><p>

```
copying path '/nix/store/q92kjbb3i01dp4hpslpq7lcn8x6p1naf-gmp-6.1.2' from 'https://cache.nixos.org'...
copying path '/nix/store/x005987xqwcb89w7103xnqr7y2m934fx-coreutils-8.30' from 'https://cache.nixos.org'...
copying path '/nix/store/sl8jnxmazmi20cavdmz12nb79s3phi4q-swift-corefoundation' from 'https://cache.nixos.org'...
copying path '/nix/store/ifdm4pslw8qh3zfqdapp6kxshfs701lc-configd-osx-10.8.5' from 'https://cache.nixos.org'...
copying path '/nix/store/0xs4x0jia6q3jfdszrnxf133njym1asp-gdbm-1.18' from 'https://cache.nixos.org'...
copying path '/nix/store/7q9djnxihh7w0hd4l8dsfrn3lmwf15cp-openssl-1.0.2p' from 'https://cache.nixos.org'...
copying path '/nix/store/6rrh0ly1n7mz2nn3px0a4jywdzsng5yq-readline-6.3p08' from 'https://cache.nixos.org'...
copying path '/nix/store/dxh6blqy4714dsjdxnsxpjrf21rrq5yr-sqlite-3.24.0' from 'https://cache.nixos.org'...
copying path '/nix/store/x7iy4sqvicc3as1hq4hd5x64kdlxxryw-python-2.7.15' from 'https://cache.nixos.org'...
/nix/store/x7iy4sqvicc3as1hq4hd5x64kdlxxryw-python-2.7.15
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=fd823ed4-3cbd-432a-b89a-8053d2a349e3-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49741&attempt_id=cf918555-be7b-4af6-bfc3-08f4d26078e8)

Attempted: python

<details><summary>Partial log (click to expand)</summary><p>

```
copying path '/nix/store/rgs0rl2nr5icn0c46396drqdf9267dj3-gcc-7.3.0-lib' from 'https://cache.nixos.org'...
copying path '/nix/store/df3vplpq00imj9yyk0jkywhc6wym0gdx-gdbm-1.18' from 'https://cache.nixos.org'...
copying path '/nix/store/ci7lk0iyf2na1ywnxn61xjrvkql8ax8w-db-5.3.28' from 'https://cache.nixos.org'...
copying path '/nix/store/w5bjx97a3gry1f9ah25lxl8gc4pmd9zc-ncurses-6.1' from 'https://cache.nixos.org'...
copying path '/nix/store/nlb8hykwq7jc4vpnda39civ9cvrb7ymq-openssl-1.0.2p' from 'https://cache.nixos.org'...
copying path '/nix/store/dc1m33pqqbb6nfpnnf3w4h0v5b6rcvw6-readline-6.3p08' from 'https://cache.nixos.org'...
copying path '/nix/store/2dbjpfa4s4c51h70vbzrcbl2krw3c3z4-zlib-1.2.11' from 'https://cache.nixos.org'...
copying path '/nix/store/08x3hgx0izn4v0fsngchfmbqwlx4jqmw-sqlite-3.24.0' from 'https://cache.nixos.org'...
copying path '/nix/store/3wdvsid5yhvx5db3rb00jn4gwrdhqs54-python-2.7.15' from 'https://cache.nixos.org'...
/nix/store/3wdvsid5yhvx5db3rb00jn4gwrdhqs54-python-2.7.15
```
</p></details>

FRidh committed:
python.pkgs.simplejson: disable tests with python 3

Commit:
python.pkgs.click: rebase fix-paths.patch

Commit:
python.pkgs.pytest_xdist: add filelock to checkInputs

Commit:
python.pkgs.virtualenv: rebase virtualenv-change-prefix.patch

Commit:
python3.pkgs.multidict: add pytestcov to checkInputs

Commit:
python: hbmqtt: 0.9.2 -> 0.9.4

Commit:
python.pkgs.subunit: move testscenarios to checkInputs

Commit:
python.pkgs.testrepository: correctly use buildInputs, checkInputs

Commit:
python.pkgs.cached-property: 1.4.2 -> 1.5.1

Commit:
python.pkgs.pyflakes: disable tests

FRidh committed:
python: botocore: 1.12.37 -> 1.12.42

FRidh committed:
python: DataModelDict: 0.9.4 -> 0.9.5

FRidh committed:
python: dask: 0.20.0 -> 0.20.1

FRidh committed:
python: dependency-injector: 3.14.0 -> 3.14.2

FRidh committed:
python: cement: 3.0.0 -> 3.0.2

FRidh committed:
python: cufflinks: 0.14.5 -> 0.14.6

FRidh committed:
python: branca: 0.3.0 -> 0.3.1

FRidh committed:
python: cypari2: 2.0.0 -> 2.0.1

FRidh committed:
python: dulwich: 0.19.6 -> 0.19.8

FRidh committed:
python: hgsvn: 0.3.11 -> 0.3.15

FRidh committed:
python: hbmqtt: 0.9.4 -> 0.9.5

FRidh committed:
python: keyring: 16.0.0 -> 16.0.2

FRidh committed:
python: fs: 2.1.1 -> 2.1.2

FRidh committed:
python: lz4: 2.1.1 -> 2.1.2

FRidh committed:
python: histbook: 1.2.4 -> 1.2.5

FRidh committed:
python: django-reversion: 3.0.1 -> 3.0.2

FRidh committed:
python: mozrunner: 7.0.1 -> 7.0.2

FRidh committed:
python: imbalanced-learn: 0.4.2 -> 0.4.3

FRidh committed:
python: httpretty: 0.9.5 -> 0.9.6

FRidh committed:
python: pysrim: 0.5.9 -> 0.5.10

FRidh committed:
python: psycopg2: 2.7.5 -> 2.7.6

FRidh committed:
python: pbr: 5.1.0 -> 5.1.1

FRidh committed:
python: nipype: 1.1.4 -> 1.1.5

FRidh committed:
python: pymongo: 2.9.1 -> 2.9.5

FRidh committed:
python: msgpack-numpy: 0.4.4.1 -> 0.4.4.2

FRidh committed:
python: plotly: 3.4.0 -> 3.4.1

FRidh committed:
python: numpy: 1.15.3 -> 1.15.4

FRidh committed:
python: paste: 3.0.3 -> 3.0.4

FRidh committed:
python: nodeenv: 1.3.2 -> 1.3.3

FRidh committed:
python: moto: 1.3.6 -> 1.3.7

FRidh committed:
python: pylint: 1.9.2 -> 1.9.3

FRidh committed:
python: pyramid: 1.10 -> 1.10.1

FRidh committed:
python: pytest-xdist: 1.24.0 -> 1.24.1

FRidh committed:
python: requests: 2.20.0 -> 2.20.1

FRidh committed:
python: python-hosts: 0.4.1 -> 0.4.5

FRidh committed:
python: pytest-sugar: 0.9.1 -> 0.9.2

FRidh committed:
python: python-json-logger: 0.1.9 -> 0.1.10

FRidh committed:
python: regex: 2018.11.03 -> 2018.11.07

FRidh committed:
python: responses: 0.10.2 -> 0.10.3

FRidh committed:
python: SQLAlchemy: 1.2.13 -> 1.2.14

FRidh committed:
python: virtualenv-clone: 0.2.5 -> 0.2.6

gebner commented on pkgs/development/python-modules/shippai/default.nix:
```diff
@@ -3,11 +3,11 @@
 buildPythonPackage rec {
   pname = "shippai";
   # Please make sure that vdirsyncer still builds if you update this package.
-  version = "0.2.4";
```
Does vdirsyncer still build?

Commit:
python.pkgs.pyjwt: disable tests

They are broken due to a pytest upgrade.

dotlambda commented on pkgs/development/python-modules/sphinx/default.nix:
```diff
@@ -25,7 +25,7 @@
 
 buildPythonPackage rec {
-  pname = "sphinx";
+  pname = "Sphinx";
```
@Synthetica9 Was there a particular reason for the change? It causes sphinx to fail to build so I reverted it.

Synthetica9 replied:
See #50269

gebner reviewed (changes requested):
I just tried to build `vdirsyncer` (now that rustc is in the cache), and it fails to build because of the `shippai` update.  Could you please revert the `shippai` update?

dotlambda commented:
@ryantm Most of these conflicts are due to your bot upgrading python packages. I think you at least shouldn't do python upgrades when the package has already been upgraded in one of the staging/staging-next/python-unstable branches.

FRidh commented:
@dotlambda it's something worth considering, but I am not sure whether it is worth spending time on. These kind of merge conflicts are at least trivial to resolve, though the hash does always look differently.

dotlambda commented:
Alternatively, the update script could use long hashes for `fetchPypi`.

ryantm commented:
@dotlambda I check master and staging currently. I'll add staging-next and python-unstable.

FRidh committed:
python: Wand: 0.4.4 -> 0.4.5

FRidh committed:
python: autopep8: 1.4.2 -> 1.4.3

FRidh committed:
python: alembic: 1.0.2 -> 1.0.3

FRidh committed:
python: dask: 0.20.1 -> 0.20.2

FRidh committed:
python: backports.shutil_which: 3.5.1 -> 3.5.2

FRidh committed:
python: distributed: 1.24.1 -> 1.24.2

FRidh committed:
python: coverage: 4.5.1 -> 4.5.2

FRidh committed:
python: awkward: 0.4.3 -> 0.4.4

FRidh committed:
python: gmusicapi: 11.0.3 -> 11.0.4

FRidh committed:
python: django-redis: 4.9.0 -> 4.9.1

FRidh committed:
python: botocore: 1.12.42 -> 1.12.48

FRidh committed:
python: gin-config: 0.1.1.2 -> 0.1.2

FRidh committed:
python: Fiona: 1.8.0 -> 1.8.2

FRidh committed:
python: deform: 2.0.6 -> 2.0.7

FRidh committed:
python: Faker: 0.9.2 -> 0.9.3

FRidh committed:
python: dulwich: 0.19.8 -> 0.19.9

FRidh committed:
python: homeassistant_pyozw: 0.1.0 -> 0.1.1

FRidh committed:
python: pgspecial: 1.11.2 -> 1.11.3

FRidh committed:
python: libagent: 0.12.0 -> 0.12.1

FRidh committed:
python: paperspace: 0.0.11 -> 0.0.12

FRidh committed:
python: neovim: 0.3.0 -> 0.3.1

FRidh committed:
python: notebook: 5.7.0 -> 5.7.2

FRidh committed:
python: openpyxl: 2.5.9 -> 2.5.10

FRidh committed:
python: inflect: 1.0.1 -> 1.0.2

FRidh committed:
python: psycopg2: 2.7.6 -> 2.7.6.1

FRidh committed:
python: node-semver: 0.5.0 -> 0.5.1

FRidh committed:
python: hupper: 1.4 -> 1.4.1

FRidh committed:
python: pex: 1.5.2 -> 1.5.3

FRidh committed:
python: responses: 0.10.3 -> 0.10.4

FRidh committed:
python: wheel: 0.32.2 -> 0.32.3

FRidh committed:
python: pytest-localserver: 0.4.1 -> 0.4.2

FRidh committed:
python: pyannotate: 1.0.6 -> 1.0.7

FRidh committed:
python: Unidecode: 1.0.22 -> 1.0.23

FRidh committed:
python: pyhomematic: 0.1.51 -> 0.1.52

FRidh committed:
python: ropper: 1.11.8 -> 1.11.10

FRidh committed:
python: whitenoise: 4.1 -> 4.1.2

FRidh committed:
python: pyrsistent: 0.14.5 -> 0.14.6

FRidh committed:
python: sseclient: 0.0.20 -> 0.0.22

FRidh committed:
python: WebOb: 1.8.3 -> 1.8.4

FRidh committed:
python: reportlab: 3.5.9 -> 3.5.10

FRidh committed:
python: QtAwesome: 0.5.2 -> 0.5.3

FRidh commented:
@ryantm not python-unstable, there is no guarantee that changes in there go in.

FRidh committed:
python.pkgs.importlib-resources: init at 1.0.2

FRidh committed:
python.pkgs.importlib-metadata: init at 0.6

FRidh committed:
python: setuptools: 40.5.0 -> 40.6.2

FRidh committed:
python.pkgs.bootstrapped-pip: setuptools: 40.4.3 -> 40.6.2

FRidh committed:
fixup zope_testrunner

FRidh committed:
fixup zope_configuration

FRidh committed:
fixup aws-xray-sdk

FRidh committed:
fixup hmmlearn

FRidh committed:
fixup google_apputils

FRidh committed:
fixup pyutil

FRidh committed:
fixup zbase32

FRidh committed:
python.pkgs.flexmock: init at 0.10.2

FRidh committed:
python.pkgs.iocapture: init at 0.1.2

FRidh committed:
fixup iocapture

FRidh committed:
fixup argh

FRidh committed:
fixup flask-restplus

FRidh committed:
python.pkgs.cytoolz: disable a test

FRidh committed:
python.pkgs.m2crypto: fix build

FRidh committed:
awscli: fix build

dtzWill committed:
pythonPackages.cython: 0.29 -> 0.29.1

https://cython.readthedocs.io/en/latest/src/changes.html#id2
(link not stable/permanent as new versions release)

Commit:
python.pkgs.ddt: fix tests

dotlambda commented:
@FRidh Do you have an idea what to do about the dateparser test failures?

FRidh committed:
python: boto3: 1.7.84 -> 1.9.36

FRidh committed:
python.pkgs.acoustics: broken

Commit:
python.pkgs.datadog: 0.23.0 -> 0.25.0

costrouc committed:
pythonPackages.livereload: 2.5.1 -> 2.6.0

costrouc committed:
pythonPackages.chalice: init at 1.6.1

costrouc committed:
pythonPackages.PyLTI: refactor add chalice as dependency fix build

costrouc committed:
pythonPackages.nose-randomly: init at 1.2.5

costrouc committed:
pythonPackages.httpretty: 0.8.10 -> 0.9.6

nixos-discourse commented:
This pull request has been mentioned on **Nix community**. There might be relevant details there:

https://discourse.nixos.org/t/where-to-find-current-failed-hydra-builds-in-master/1545/4

costrouc committed:
pythonPackages.factory_boy: 2.6.1 -> 2.11.1

costrouc committed:
pythonPackages.Quandl: 3.2.1 -> 3.4.5

costrouc committed:
pythonPackages.iocapture: refactor meta attribute license

costrouc committed:
pythonPackages.factory_boy: refactor remove fake_factory dependency

costrouc committed:
pythonPackages.fake_factory: refactor disable tests (for good reason)

costrouc committed:
pythonPackages.faker: refactor remove pytest requirements

costrouc committed:
pythonPackages.tifffile: 2018.10.18 -> 0.15.1

Downgrade package. 2018.XX.XX are not official releases? Fixes
dask-image build as well.

costrouc committed:
python3Packages.gidgethub: refactor fix broken package

costrouc committed:
pythonPackages.jwcrypto: init at 0.6.0 dependency for broken github3_py

costrouc committed:
pythonPackages.github3_py: refactor fix broken package

costrouc committed:
pythonPackages.lz4 refactor fix broken build

costrouc committed:
pythonPackages.Wand: refactor fix broken package

costrouc committed:
pythonPackages.altair: refactor fix broken python package

costrouc committed:
pythonPackages.asyncio: refactor disable for anything but 34

costrouc committed:
pythonPackages.awkward: 0.4.4 -> 0.5.2 fix broken package

costrouc committed:
pythonPackages.branca: refactor fix broken python package

Commit:
python.pkgs.zope-deferredimport: init at 4.3

Commit:
python.pkgs.zope-hookable: init at 4.2.0

Commit:
python.pkgs.zope_component: update propagatedBuildInputs

Commit:
python.pkgs.h11: 0.7.0 -> 0.8.1

Commit:
python.pkgs.eventlet: update propagatedBuildInputs

Commit:
python.pkgs.forbiddenfruit: disable tests because they are missing in tarball

Commit:
python.pkgs.paste: 3.0.4 -> 3.0.5

dotlambda commented:
@FRidh I think there are more failing packages on master than on python-unstable. What's holding us back from merging?

Commit:
python: pylint: 2.1.1 -> 2.2.2

Commit:
python.pkgs.pylint: disable broken tests

fixes https://github.com/NixOS/nixpkgs/issues/51394

Commit:
python.pkgs.gpsoauth: 0.2.0 -> 0.4.1

marsam committed:
pythonPackages.testfixtures: 6.2.0 -> 6.3.0

Commit:
python.pkgs.natsort: 5.4.1 -> 5.5.0

Commit:
python.pkgs.Nikola: apply patch for broken test

Commit:
python.pkgs.webtest: fix build

Commit:
python.pkgs.pyacoustid: remove obsolete patch

Commit:
python.pkgs.sphinx_rtd_theme: add sphinx to propagatedBuildInputs

Commit:
python.pkgs.whisper: fix tests

Commit:
python.pkgs.zconfig: remove obsolete patch

Commit:
python.pkgs.zodb: remove obsolete patches and fix tests

dotlambda commented:
I just rebased on staging and believe we should merge if Hydra doesn't show any major breakage.

dotlambda commented:
I just rebased on staging and believe we should merge if Hydra doesn't show any major breakage.

Commit:
python.pkgs.python-jose: 2.0.2 -> 3.0.1

Commit:
python.pkgs.moto: allow newer versions of python-jose

Mic92 commented:
When I have other minor python library updates that depend on updates here, should I just push them to the branch here as well?

dotlambda commented:
> When I have other minor python library updates that depend on updates here, should I just push them to the branch here as well?

Sounds like a sensible thing to do. After you're done with that, we should *definitely* merge this PR!

Mic92 commented:
@dotlambda you go ahead with merging. I will add it later.

dotlambda commented:
@Mic92 I prefer to let @FRidh do that.

dotlambda commented:
@GrahamcOfBorg eval
